blog/_posts/2022-04-08-scala-3.1.2-released.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.2 released!
+---
+
+We are happy to announce the release of Scala 3.1.2. This new release brings a few significant improvements awaited by the community.
+
+## Configurable Scala output version
+
+First and foremost, after months of brainstorming and testing, we are bringing an experimental possibility to configure the Scala output version. This means that the compiler can now generate TASTy files and classfiles in a format used by an older minor version of Scala (which can currently be only 3.0.x) effectively specifying the minimal version of the compiler that is able to read your compiled code.
+
+If you are a library maintainer, you may have faced a dilemma, should you update the Scala version to 3.1 or stay on 3.0. On the one hand, updating would bring you all bug fixes and enhancements from the newest version of the compiler. On the other, all new releases of your library would be unusable for all the users that are, for some reason, stuck on Scala 3.0.
+
+With Scala 3.1.2, this tradeoff is no more. Now you can set the Scala output to previous language minor version. Using 3.1.2 to compile your library, you can set your output to Scala 3.0. Then your library can be used by code compiled with Scala 3.0, 3.1, or any future version. All produced TASTy files and classfiles will be compatible with Scala 3.0 and newer.
+
+While the flag itself is not marked as experimental, its role in the broader ecosystem of scala libraries is yet to be determined. We are eagerly awaiting feedback from the maintainers of the libraries. Hearing your opinions will allow us to choose the right strategy for supporting code compiled for previous scala versions so that we can find a balance between language development and burden on both compiler users and maintainers.
```

Comment:
s/scala/Scala/  (twice)

Reply:
When reading this, it is not clear to me what type of feedback you would like to get from the community.